pr_agent/settings/configuration.toml

```diff
@@ -230,6 +238,14 @@ push_commands = [
     "/describe",
     "/review --pr_reviewer.num_code_suggestions=0",
 ]
+# a list of regular expressions to match against the PR title to ignore the PR agent
+ignore_mr_title = []
+# target branches to ignore from PR agent when an MR is created
+ignore_mr_target_branches = []
+# source branches to ignore from PR agent when an MR is created
+ignore_mr_source_branches = []
+# labels to ignore from PR agent when an MR is created
+ignore_mr_labels = []
```

Comment:
@paolomainardi remove this, via the [config]
I want to have unified params. they are easier to maintain, track an use

Reply:
Ooops, it's a leftover, I'll remove it.